### ISSUE TITLE
Disable the GitHub Wiki if its not being used

### ISSUE THREAD

**Issue by Foorack** [EFForg/https-everywhere#6307]

**Comment by jeremyn** [comment]
I agree that the wiki should be disabled if it's not being used. But, I think we could use the wiki as a place for the list of sites in issue #3069, as a place to just add sites that someone thinks needs attention, to be periodically reviewed by volunteers.

(Actually my dream is that the EFF hosts a site outside of GitHub where non-technical people can easily submit sites for review, but that would be a tricky project to do correctly.)

**Comment by Foorack** [comment]
"I think we could use the wiki as a place for the list of sites in issue #3069"
Loving that idea!! :heart: 

"submit sites for review"
Easily submit domains or rules?

**Comment by jeremyn** [comment]
Just domains, not rules. The goal is to make it as easy as possible for non-technical HTTPS Everywhere users to let technical volunteers know some site needs attention. An example user could be a non-STEM student at the community college in pull request #6279. The UI for the site would look something like https://hstspreload.appspot.com/ .

**Comment by jeremyn** [comment]
Or another idea, users could tweet domains using some hashtag: https://blogs.windows.com/msedgedev/2016/08/11/edgebug-twitter/ .

**Comment by Foorack** [comment]
> users could tweet domains using some hashtag:

I like the idea as I am a Twitter user, however, not everyone have a Twitter account and I think it will be difficult to list and mark which are done.

What about using Google Forms and having it being saved in a Spreadsheet? Contributors could mark domains with its status and the spreadsheet would be public for read-access. Thoughts?

**Comment by Foorack** [comment]
Actually, this issue is regarding the wiki. I will create another issue for the UI idea.

**Comment by jeremyn** [comment]
Another issue is fine. I'll respond to your forms comment there.

As far as this issue goes, I again agree that if the GitHub wiki is not used, it should be turned off. If it becomes used to track domains or something, it can be reactivated.

**Comment by Foorack** [comment]
@jeremyn I'm now wondering if it should be used to track domains immediately or if it should be turned off and then someone has to create an issue regarding "Using the wiki to track domains".

**Comment by jeremyn** [comment]
I'd like to get input from one of the EFF people/"high-ranking" volunteers before we start doing stuff in the wiki.

**Comment by Foorack** [comment]
@jeremyn Ah yes, it wasn't meant to sound like I suggested we start doing it immediately without approval. I updated my

**Comment by jeremyn** [comment]
Sure. Now we just wait and see.

**Comment by jeremyn** [comment]
Okay so a month later: I don't like the idea of using the wiki to track domains that need attention. I prefer one issue per (connected set of) domain(s). Some reasons:

1. Wiki editing requires limited access which is a bottleneck.
2. Wikis tend to become weird, unmaintained, and slightly embarrassing.
3. An issue allows for easier back-and-forth discussion with someone who presumably knows something about that site, if needed. For example if the site requires an account, the issue submitter might have an account they can test with. At the very least they are probably familiar with the language the site uses.
4. Adding new domains can be "easy win" issues for volunteers. Some might feel more satisfaction in working to close an issue than working to check off something in a list.
5. Issues count as GitHub contributions, which might encourage people to submit them.
5. We can't use GitHub tooling such as labels on a per-site basis in the wiki or a collection issue like #3069. Imagine if we had some version of the Alexa popularity labeler that worked for issues.

I know @terrorist96 means well with #3069, and it's great that they are making these domains visible and tracking progress and doing that work, but it's sort of an anti-pattern.

**Comment by Hainish** [comment]
I've disabled the wiki for now.  We can re-enable it if needed later.  I understand there is a separate discussion around maintenance of the list of desired rulesets, which seems separate (though related) to this issue.  We should take this discussion to a separate issue.

I'm going to close this issue for now.

**Comment by jeremyn** [comment]
@Hainish That's fine with me.

Not to reopen the discussion, but I want to add that my opinion has changed since my previous comment https://github.com/EFForg/https-everywhere/issues/6307#issuecomment-247493777 . I think minimizing the number of open issues is really important, so I support something like #3069 instead of individual issues like I said in my earlier comment.
